.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: "Run tests"
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Not sure how it works but maybe we can use the same images?

Reply:
I'll have a look into it